src/pages/MyNotes/MyNotes.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@mui/material';
+
+import { NotesList } from 'pages/MyNotes';
+import { ActiveNote, CreateNoteContainer } from 'components';
```

Comment:
if you add index.ts in CreateNote folder with export from CreateNoteContainer, you can use this way of imports
```suggestion
import { ActiveNote, CreateNote } from 'components';
```